pageserver/src/tenant/delete.rs

```diff
@@ -286,7 +286,7 @@ impl DeleteTenantFlow {
     ) -> Result<(), DeleteTenantError> {
         span::debug_assert_current_span_has_tenant_id();
 
-        pausable_failpoint!("tenant-delete-before-run");
+        crate::failpoint_support::pausable_failpoint!("tenant-delete-before-run");
```

Comment:
nit; Why change all the existing failpoints to qualified import? 
I think mid-term we want our own async failpoints library anyways, if we keep the unqualified usage, we'll just need to change the import.

Reply:
I thought it was a good idea in the beginning. Can switch them back before merging.